src/app/create-article/create-article.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { UpdateDataService } from '../update-data.service';
+
+@Component({
+  selector: 'app-create-article',
+  templateUrl: './create-article.component.html',
+  styleUrls: ['./create-article.component.css']
+})
+
+export class CreateArticleComponent implements OnInit {
+  constructor(private channelService: UpdateDataService) { 
+    this.channelService.updateFlag(0);
```

Comment:
`updateFlag` method name should be renamed to reflect its actual functionality

Reply:
**Changes implemented**